pkgs/development/libraries/gtkmm/3.x.nix

```diff
@@ -11,7 +11,7 @@ stdenv.mkDerivation rec {
 
   outputs = [ "out" "dev" ];
 
-  nativeBuildInputs = [ pkg-config meson ninja python3 ];
+  nativeBuildInputs = [ pkg-config meson ninja python3 glib gdk-pixbuf ];
```

Comment:
`gdk-pixbuf` is suspect as well.

Reply:
not suspect

```
gtkmm-aarch64-unknown-linux-gnu> FAILED: demos/gtk-demo/demo-resources.c
gtkmm-aarch64-unknown-linux-gnu> /nix/store/f9fbghzfw6xdy87bh212pwnc03n9mlrw-glib-2.72.3-dev/bin/glib-compile-resources ../demos/gtk-demo/dem
o.gresource.xml --sourcedir /build/gtkmm-3.24.6/demos/gtk-demo --sourcedir ../demos/gtk-demo --internal --generate --target demos/gtk-demo/de
mo-resources.c --dependency-file demos/gtk-demo/demo-resources.c.d
gtkmm-aarch64-unknown-linux-gnu> ../demos/gtk-demo/demo.gresource.xml: to-pixdata preprocessing requested, but GDK_PIXBUF_PIXDATA is not set,
 and gdk-pixbuf-pixdata is not in PATH.
 ```

Reply:
Hmm, might be nice to mention that it is for `gdk-pixbuf-pixdata` in a comment, it is pretty rare so might be accidentally removed.

Reply:
added